ExperimentTwitter/UserHeaderView.h

```diff
@@ -0,0 +1,15 @@
+//
+//  UserHeaderView.h
+//  ExperimentTwitter
+//
+//  Created by ayur.j on 18/09/17.
+//  Copyright © 2017 ayur.j. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface UserHeaderView : UIView
+@property (strong, nonatomic, nullable) IBOutlet UIImageView *profileImageView;
+@property (strong, nonatomic, nullable) IBOutlet UILabel *userNameLabel;
+@property (strong, nonatomic, nullable) IBOutlet UILabel *handleLabel;
```

Comment:
These outlets could be in private category (i.e. .m file). To allow its text, etc. to be configured, you can expose different method(s) to configure those. 
Be very careful in exposing properties in header file. This is similar to the advice given in Java regarding public/private fields.